packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -205,6 +205,15 @@ abstract class FlutterDriver {
   }
 
   /// Waits until [finder] locates the target.
+  ///
+  /// The [finder] will wait until there is no pending frame scheduled
+  /// in the app under test before executing an action.
+  ///
+  /// See also:
+  ///
+  ///  * [FlutterDriver.runUnsynchronized], which will execute an action
+  ///  with frame sync disabled and while the app under test is undergoing
+  ///  a transition.
```

Comment:
```suggestion
  ///  * [FlutterDriver.runUnsynchronized], which will execute an action
  ///    with frame sync disabled even while frames are pending.
```

Reply:
@goderbauer I've made these changes. Will push update.

Reply:
@goderbauer  pushed this change and ready for a look.